pkg/services/publicdashboards/models/models.go

```diff
@@ -40,20 +40,39 @@ var (
 type ShareType string
 
 type PublicDashboard struct {
-	Uid                  string        `json:"uid" xorm:"pk uid"`
-	DashboardUid         string        `json:"dashboardUid" xorm:"dashboard_uid"`
-	OrgId                int64         `json:"-" xorm:"org_id"` // Don't ever marshal orgId to Json
+	Uid          string    `json:"uid" xorm:"pk uid"`
+	DashboardUid string    `json:"dashboardUid" xorm:"dashboard_uid"`
+	OrgId        int64     `json:"-" xorm:"org_id"` // Don't ever marshal orgId to Json
+	AccessToken  string    `json:"accessToken" xorm:"access_token"`
+	CreatedBy    int64     `json:"createdBy" xorm:"created_by"`
+	UpdatedBy    int64     `json:"updatedBy" xorm:"updated_by"`
+	CreatedAt    time.Time `json:"createdAt" xorm:"created_at"`
+	UpdatedAt    time.Time `json:"updatedAt" xorm:"updated_at"`
+	//config fields
 	TimeSettings         *TimeSettings `json:"timeSettings" xorm:"time_settings"`
-	IsEnabled            bool          `json:"isEnabled" xorm:"is_enabled"`
-	AccessToken          string        `json:"accessToken" xorm:"access_token"`
-	AnnotationsEnabled   bool          `json:"annotationsEnabled" xorm:"annotations_enabled"`
 	TimeSelectionEnabled bool          `json:"timeSelectionEnabled" xorm:"time_selection_enabled"`
+	IsEnabled            bool          `json:"isEnabled" xorm:"is_enabled"`
+	AnnotationsEnabled   bool          `json:"annotationsEnabled,omitempty" xorm:"annotations_enabled"`
```

Comment:
The `omitempty` option here is preventing returning the value when is `false` in the PATCH and POST response.

Reply:
yes, you're right. It might have been a wrong copy paste because it doesn't make sense to have done that modification, gonna revert it